src/components/redeemPosition/SelectPosition/index.tsx

```diff
@@ -64,15 +49,10 @@ export const SelectPosition = () => {
           <Textfield
             disabled={true}
             error={!!errors.length}
-            placeholder="Please select a position..."
+            placeholder={loading ? 'Loading...' : 'Please select a position...'}
```

Comment:
Similar scenario with the `Loading...` placeholder in Select Condition component, although this component will be replaced with a `SelectPositionModal` component

Reply:
Yeah, we'll fix it with https://github.com/gnosis/conditional-tokens-explorer/issues/139